Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM phusion/baseimage:latest
```

Comment:
I understand why you want to use this instead of `ubuntu:latest`, but this allows external people to put arbitrary code on our docker instances. I'd feel safer if we just used `ubuntu:latest` and trusted canonical.

We could alternatively build and host this baseimage ourselves on our dockerhub account, but that's more work. We can always do that in the future though, so for now I think this should be changed to `ubuntu:latest`.